conanfile.py

```diff
@@ -33,7 +33,10 @@ def build(self):
             cmake.definitions["CMAKE_CXX_STANDARD"] = "17"
             cmake.definitions["BSONCXX_POLY_USE_STD"] = True
         else:
-            self.output.fatal("Only C++17 supported for now. Support boost dependency?")
+            cmake.definitions["CMAKE_CXX_STANDARD"] = "14"
```

Comment:
Since we are using boost, should (and can) we downgrade it to C++11?

Reply:
I think we could, it looks like it works testing locally... what would be the impact of deleting the line altogether?

Reply:
I suggest we change it to `11`instead of `14`